backend/inspirehep/mailing/providers/flask_mail.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from flask import current_app
+from flask_mail import Message
+
+
+def send_email(sender, recipient, subject, body, cc=None):
+    msg = Message(
+        subject,
+        sender=sender,
+        recipients=[recipient],
+        html=body,
```

Comment:
I think you want to add `body=strip_all_html_tags(body)` (with a suitably defined function) to have a plain text version.

Reply:
Good suggestion